src/backend/web/routes/feeds.js

```diff
@@ -74,6 +75,18 @@ feeds.post('/', protect(), async (req, res) => {
   }
 });
 
+feeds.put('/cache', protectAdmin(true), async (req, res) => {
+  try {
+    await Feed.clearCache();
+    return res.status(200).json({ message: 'Cache successfully cleared' });
```

Comment:
Probably 204 vs. 200 and return nothing.

Reply:
Thanks! Was debating between the two